wrappers/gin/gin.go

```diff
@@ -33,13 +88,17 @@ func wrapGinHandler(handler gin.HandlerFunc, hostname string, relativePath strin
 		}
 		c.Keys[TracerKey] = wrapperTracer
```

Comment:
would use here c.Set (even though its the same logic)

Reply:
:+1: